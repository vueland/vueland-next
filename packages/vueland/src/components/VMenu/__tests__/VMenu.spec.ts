import { mount } from '@vue/test-utils'
import { VMenu } from '../index'

describe('VMenu', () => {
  let mountFunction: (options?: any) => any

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VMenu, { ...options })
  })

  test('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
