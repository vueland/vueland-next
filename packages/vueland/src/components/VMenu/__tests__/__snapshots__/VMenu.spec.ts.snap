// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VMenu should mount component and match snapshot 1`] = `
<div class="v-menu">
</div>
<transition-stub name="fade"
                 mode
                 appear="false"
                 persisted="false"
                 css="true"
>
</transition-stub>
`;
